src/components/Header-Restoran/Header-Restoran.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
Why does component have a different name than name of file itself

Reply:
Change name component to HeaderRestourant